### PR TITLE
Fractions HT is ready

### PR DESCRIPTION
**Pull request by JavaEz**
Viacheslav, please check.

### CODE REVIEW

File: fractions/FractionNumberImpl.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+	@Override
+	public double value() {
+		
+		return (double)divisor / divident;
+	}
+
+	@Override
+	public String toString() {
+		return this.divisor + "/" + this.divident;
```
Comment (ViacheslavButov) [inline fractions/FractionNumberImpl.java]: please do not use **this** without name conflicts

Comment (JavaEz) [reply fractions/FractionNumberImpl.java]: I got it! Ready.

File: fractions/FractionNumberImpl.java [hunk 2]
```diff
@@ -6,8 +6,8 @@
 	private int divisor; // delitel
 
 	FractionNumberImpl(int divisor, int divident) {
-		this.divisor = divisor;
-		this.divident = divident;
```
Comment (ViacheslavButov) [inline fractions/FractionNumberImpl.java]: it is not correct place, please make corrections exactly where I left my comment

Comment (JavaEz) [reply fractions/FractionNumberImpl.java]: ready.

File: fractions/FractionNumberOperationImpl.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	@Override
+	public FractionNumber mul(FractionNumber a, FractionNumber b) {
+		int resultA = a.getDivisor() * b.getDivisor();
+		int resultB = a.getDivident() * b.getDivident();
+		return new FractionNumberImpl(resultA, resultB);
+	}
+
+	@Override
+	public FractionNumber sub(FractionNumber a, FractionNumber b) {
+		int resultA = (a.getDivisor() * b.getDivident()) - (b.getDivisor() * a.getDivident());
```
Comment (ViacheslavButov) [inline fractions/FractionNumberOperationImpl.java]: java can handle operations priorities, so it is not required to use additional bracers

Comment (JavaEz) [reply fractions/FractionNumberOperationImpl.java]: ready.

**Pull request merged**